2-mandatory/1-weather-report.js

```diff
@@ -11,25 +11,29 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
-    // TODO
-}
-
+    function getTemperatureReport(cities){
+        const statement = [];
```

Comment:
Another small point - is `statement` a good name for this variable? Maybe `statements` or `weatherReports` would be a better name as this is an array which contains many statements 😄